### DISCUSSION TITLE
Wandb integration

### DISCUSSION THREAD

**Discussion by adwaykanhere** [MIC-DKFZ/nnUNet#1305] [Ideas]
Hi @FabianIsensee 

Is it possible to integrate [wandb](https://wandb.ai)/ any other ML monitoring interface into nnUNet? 
I'm interested in making this addition to the codebase but I don't know where to start and need your help.

**Comment by FabianIsensee** [comment]
Hey, 
for now I am not looking to do this because it overcomplicates things for me. Thanks for the suggestion though!
Best,
Fabian
